wpilib/tests/test_drive.py

```diff
@@ -123,16 +123,18 @@ def drive_mecanum(wpimock, halmock):
     m4.reset_mock()
     return drive
 
-
+quickStopAccumulator = 0
 def check_curvature(wpimock, drive_diff, y, rotation, isQuickTurn):
-    quickStopAccumulator = 0
+    global quickStopAccumulator
```

Comment:
why is this a global

Reply:
I changed it to act as a real accumulator for the tests. Otherwise it would get overwritten every time the function changed. None of our test cases rely on this, but I think it'd be good to allow for it in the future.

Reply:
I don't think globals in tests work like that

Reply:
really?

Reply:
hm looks like you're right. I'd have to use a fixture